shivyc/main.py

```diff
@@ -118,6 +122,12 @@ def get_arguments():
     parser.add_argument("-z-reg-alloc-perf",
                         help="display register allocator performance info",
                         dest="show_reg_alloc_perf", action="store_true")
+    parser.add_argument(
+        "-o",
+        nargs="?",
+        metavar="file",
+        help="Place output into <file>",
```

Comment:
Nitpicky: for consistency, could you make the first letter of the help string lowercase?